animation_nodes/nodes/matrix/distribute_matrices.pyx

```diff
@@ -385,6 +400,34 @@ class DistributeMatricesNode(bpy.types.Node, AnimationNode):
         count = len(spline.points)
         return spline.getDistributedMatrices(count, 0, 1, "RESOLUTION")
 
+    def execute_HexagonGrid(self, int xDiv, int yDiv, float size1, float size2, float size3):
+        cdef Py_ssize_t i, j, index
+        cdef float x, y
```

Comment:
Move declarations as close as possible to initializations. So those should be declared just before the loop.

Reply:
Done